pkg/controllers/subnetport/subnetport_controller.go

```diff
@@ -460,14 +542,23 @@ func (r *SubnetPortReconciler) CheckAndGetSubnetPathForSubnetPort(ctx context.Co
 		// subnetport is needed to be deleted.
 		_, err = r.SubnetService.GetSubnetByPath(subnetPath)
 		if err != nil {
-			log.Info("previous NSX subnet is deleted, deleting the stale subnet port", "subnetPort.UID", subnetPort.UID, "subnetPath", subnetPath)
+			log.Info("Previous NSX Subnet is deleted, deleting the stale SubnetPort", "subnetPort.UID", subnetPort.UID, "SubnetPath", subnetPath)
 			if err = r.SubnetPortService.DeleteSubnetPortById(subnetPortID); err != nil {
-				log.Error(err, "failed to delete the stale subnetport", "subnetport.UID", subnetPort.UID)
+				log.Error(err, "Failed to delete the stale SubnetPort", "subnetPort.UID", subnetPort.UID)
 				return
 			}
 		} else {
-			log.V(1).Info("NSX subnet port had been created, returning the existing NSX subnet path", "subnetPort.UID", subnetPort.UID, "subnetPath", subnetPath)
+			log.V(1).Info("NSX SubnetPort had been created, returning the existing NSX Subnet path", "subnetPort.UID", subnetPort.UID, "subnetPath", subnetPath)
+			existing = true
```

Comment:
Where is the var `existing` used?

Reply:
It is returned here and used for checking whether we need to release the port from the map recording runtime port count under creation on subnet.

https://github.com/vmware-tanzu/nsx-operator/blob/6822956cc395dc50ee52cca75d60aa6f62304804/pkg/controllers/subnetport/subnetport_controller.go#L96